WordPress/Classes/Models/Post.m

```diff
@@ -21,8 +40,29 @@ - (void)didTurnIntoFault
     self.specialType = nil;
 }
 
+- (void)didSave
+{
+    [super didSave];
+    if ([self isDeleted]) {
```

Comment:
Note here: *isDeleted* will only return true if the object is deleted, and the MOC wasn't saved. 

Meaning that if the object isDeleted, and the MOC is saved, this method may return false (even when the object was effectively deleted).

In that case, addressing any of the MO's properties may trigger a NSObjectInaccessibleException. Safest way to check this is by means of the *existingObjectWithId* selector ([reference here](http://stackoverflow.com/questions/4340445/how-can-i-tell-whether-an-nsmanagedobject-has-been-deleted/7896369#7896369))